jupyterlab/staging/templates/Untitled.ipynb

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Would you mind removing this file that accidentally slip in?

Reply:
done :+1: